### PR TITLE
Add Mailjet API to Business

### PR DESCRIPTION
**Pull request by TLOWAC**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [ ] I have searched the repository for any relevant issues or pull requests
- [ ] Any category I am creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ API | Description | Auth | HTTPS | CORS |
 | [MailboxValidator](https://www.mailboxvalidator.com/api-email-free) | Validate email address to improve deliverability | `apiKey` | Yes | Unknown |
 | [Mailchimp](https://mailchimp.com/developer/) | Send marketing campaigns and transactional mails | `apiKey` | Yes | Unknown |
 | [mailgun](https://www.mailgun.com/) | Email Service | `apiKey` | Yes | Unknown |
-| [mailjet](https://www.mailjet.com/) |  Marketing email can be sent and mail templates made in MJML or HTML can be sent using API. | `apiKey` | Yes | Unknown |
+| [mailjet](https://www.mailjet.com/) |  Marketing email can be sent and mail templates made in MJML or HTML can be sent using API | `apiKey` | Yes | Unknown |
```
Comment (pawelborkar) [inline README.md:270]:
```suggestion
| [mailjet](https://www.mailjet.com/) | Marketing email can be sent and mail templates made in MJML or HTML can be sent using API | `apiKey` | Yes | Unknown |
```

Comment (pawelborkar) [reply README.md:270]: each segment must start and end with exactly 1 space

**Review by pawelborkar — CHANGES_REQUESTED** [review]
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

File: README.md [hunk 2]
```diff
@@ -267,6 +267,11 @@ API | Description | Auth | HTTPS | CORS |
 | [MailboxValidator](https://www.mailboxvalidator.com/api-email-free) | Validate email address to improve deliverability | `apiKey` | Yes | Unknown |
 | [Mailchimp](https://mailchimp.com/developer/) | Send marketing campaigns and transactional mails | `apiKey` | Yes | Unknown |
 | [mailgun](https://www.mailgun.com/) | Email Service | `apiKey` | Yes | Unknown |
+<<<<<<< Updated upstream
+| [mailjet](https://www.mailjet.com/) |  Marketing email can be sent and mail templates made in MJML or HTML can be sent using API | `apiKey` | Yes | Unknown |
+=======
+| [mailjet](https://www.mailjet.com/) | Marketing email can be sent and mail templates made in MJML or HTML can be sent using API. | `apiKey` | Yes | Unknown |
```
Comment (pawelborkar) [inline README.md:273]: description should not end with .

Comment (pawelborkar) [reply README.md:273]:
```suggestion
| [mailjet](https://www.mailjet.com/) | Marketing email can be sent and mail templates made in MJML or HTML can be sent using API | `apiKey` | Yes | Unknown |
```

**Review by pawelborkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (TLOWAC) [reply README.md]: I fixed it !

**Comment by pawelborkar** [comment]
Thanks for the contribution @TLOWAC

**Pull request merged**